Makefile

```diff
@@ -34,10 +38,10 @@ start-frontend:
 run:
 	@echo "Running the app..."
 	@mkdir -p logs
-	@rm -f logs/pipe
-	@mkfifo logs/pipe
-	@cat logs/pipe | (make start-backend) &
-	@echo 'test' | tee logs/pipe | (make start-frontend)
+	@poetry run nohup uvicorn opendevin.server.listen:app --port $(BACKEND_PORT) > logs/backend_$(shell date +'%Y%m%d_%H%M%S').log 2>&1 &
+	@echo "Waiting for the backend to start..."
+	@until nc -z localhost $(BACKEND_PORT); do sleep 0.1; done
+	@cd frontend && npm run start -- --port $(FRONTEND_PORT)
```

Comment:
Is this change intentional? Current head of main uses the mkfifo style rather than nohup.